ethereum/pow/merkletree/src/main/java/tech/pegasys/teku/ethereum/pow/merkletree/DepositTreeSnapshot.java

```diff
@@ -43,6 +51,11 @@ public Bytes32 getExecutionBlockHash() {
     return executionBlockHash;
   }
 
+  public tech.pegasys.teku.spec.datastructures.eth1.DepositTreeSnapshot asSerializable() {
+    return tech.pegasys.teku.spec.datastructures.eth1.DepositTreeSnapshot.SSZ_SCHEMA.create(
+        finalized, UInt64.valueOf(deposits), executionBlockHash);
+  }
```

Comment:
I don't think we want to create an SSZ version of this at this stage.  We can just write it out with a custom serialiser like we do with `DepositsFromBlockEventSerializer`.